.travis.yml

```diff
@@ -1,13 +1,16 @@
 # Run tests run on travis-ci after each commit and before each merge
 
 language: java
+dist: trusty
```

Comment:
None, I think the DIST is required. It's actually in the gradle clean up commit @grimreaper had, so I'm not sure why he's against this...